src/App.tsx

```diff
@@ -1,27 +1,35 @@
-import * as React from 'react';
 import { Menu } from './navigation';
 import { useLocation } from 'react-router-dom';
+import ErrorPage from './Error';
+import VetList from './components/VetList';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Root from './Root';
+import OwnerList from './components/OwnerList';
 
 export default function App() {
-  const location = useLocation();
+  // TODO: add ErrorPage to Routes
+
+  //const location = useLocation();
 
   return (
+    <>
     <div>
-      <Menu name={location.pathname} />
+      <Router>
```

Comment:
Hmm I think this is the legacy approach to declare Router and Routes, while I admit that it could work, but based on [React Router modern guide](https://reactrouter.com/en/main/start/tutorial#jsx-routes), it seems it's no more the recommended way. 

In order to keep the new app well updated, and to be in sync with the docs, consider using `createRoutesFromElements()`.

Reply:
I'll create a separate issue for that.